Zeitpunkt: 2023-08-17 14:25:51,222
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: filter_np_morphology() got an unexpected keyword argument 'morph_dict'
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:26:16,045
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: filter_np_morphology() got an unexpected keyword argument 'morph_dict'
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:27:17,754
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: filter_np_morphology() got an unexpected keyword argument 'np_data'
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:27:36,422
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: list index out of range
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:28:32,613
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: list index out of range
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:29:18,872
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: list index out of range
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:29:42,647
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: list index out of range
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 14:30:40,010
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: list index out of range
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 80
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:49:15,070
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: NpCongruency.__init__() missing 1 required positional argument: 'np_article'
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:51:28,611
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:51:34,063
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:53:34,417
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:54:55,124
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:55:23,742
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: name 'np_congruency' is not defined
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 63
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:55:52,545
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: 'dict' object is not callable
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:56:02,051
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:56:08,720
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: 'dict' object is not callable
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:56:44,877
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: 'dict' object is not callable
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:56:55,785
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:57:08,309
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:58:02,388
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:58:10,389
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:58:32,245
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: name 'np_data' is not defined
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:58:51,976
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:59:27,470
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 16:59:48,882
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:00:10,045
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: simple_congruency_check() takes 3 positional arguments but 4 were given
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:02:16,649
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:02:29,094
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:02:38,329
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:03:21,660
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:04:18,111
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: local variable 'np_data' referenced before assignment
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:05:47,555
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: NpCongruency.__simple_congruency() takes 2 positional arguments but 3 were given
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

Zeitpunkt: 2023-08-17 17:09:12,139
Loggername: root
LogLevel: ERROR
Benutzerdefiniert: Die Analyse konnte nicht durchgefuehrt werden.
Nachricht: NpCongruency.__simple_congruency() takes 2 positional arguments but 3 were given
Modul: app_congruential_analysis
Funktion: run_simple_analysis
Linie: 64
Dateipfad: /Users/christopherchandler/repo/Python/De_NP_Kongru/kongru/api_nlp/congruential_analysis/app_congruential_analysis.py

